test/deploy/kubernetes/runtime-pods/components_data_node2.yaml

```diff
@@ -0,0 +1,170 @@
+  selector:
+    matchLabels:
+      app: data-node2
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
+  minReadySeconds: 5
+  template:
+    metadata:
+      labels:
+        app: data-node2
+    spec:
+      hostname: data-node2
+      nodeSelector:
+        node-name: node2
+      volumes:
+      - name: solution-config
+        configMap:
+          name: solution-config
+      - name: node-id
+        configMap:
+          name: data-node2-id
+      - name: shared-node2
+        persistentVolumeClaim:
+          claimName: shared-node2
+      - name: local-node2
+        persistentVolumeClaim:
+          claimName: local-node2
+      - name: sdb
+        persistentVolumeClaim:
+          claimName: dev-sdb-node2
+      - name: sdc
+        persistentVolumeClaim:
+          claimName: dev-sdc-node2
+      - name: sdd
+        persistentVolumeClaim:
+          claimName: dev-sdd-node2
+      - name: sde
+        persistentVolumeClaim:
+          claimName: dev-sde-node2
+      - name: sdf
+        persistentVolumeClaim:
+          claimName: dev-sdf-node2
+      - name: sdg
+        persistentVolumeClaim:
+          claimName: dev-sdg-node2
+      - name: sdh
+        persistentVolumeClaim:
+          claimName: dev-sdh-node2
+      - name: sdi
+        persistentVolumeClaim:
+          claimName: dev-sdi-node2
+      containers:
+      - name: cortx-hax
+        image: ghcr.io/seagate/cortx-all:2.0.0-latest-custom-ci
+        imagePullPolicy: IfNotPresent
+        args:
+        - /bin/sh
+        - -c
+        - set -x;
+          /opt/seagate/cortx/hare/bin/hare_setup start --config yaml:///etc/cortx/cluster.conf;
+          sleep infinity;
+        volumeMounts:
+        - name: solution-config
+          mountPath: /etc/cortx/solution
+        - name: node-id
+          mountPath: /etc/cortx/solution/node
+        - name: shared-node2
+          mountPath: /share
+        - name: local-node2
+          mountPath: /etc/cortx
+      - name: cortx-confd
+        image: ghcr.io/seagate/cortx-all:2.0.0-latest-custom-ci
+        imagePullPolicy: IfNotPresent
+        args:
+        - /bin/sh
+        - -c
+        - set -x;
+          /opt/seagate/cortx/motr/bin/motr_setup start --config yaml:///etc/cortx/cluster.conf --services confd;
+          sleep infinity;
+        volumeMounts:
+        - name: solution-config
+          mountPath: /etc/cortx/solution
+        - name: node-id
+          mountPath: /etc/cortx/solution/node
+        - name: shared-node2
+          mountPath: /share
+        - name: local-node2
+          mountPath: /etc/cortx
+      - name: cortx-ios1
+        image: ghcr.io/seagate/cortx-all:2.0.0-latest-custom-ci
+        imagePullPolicy: IfNotPresent
+        args:
+        - /bin/sh
+        - -c
+        - set -x;
+          /opt/seagate/cortx/motr/bin/motr_setup start --config yaml:///etc/cortx/cluster.conf --services ioservice --idx 1;
+          sleep infinity;
```

Comment:
Are we sure to call `sleep infinity` across all pods?

Reply:
It's there in the services framework also, so following the same. Also As we are mocking the upgrade flow, this throws an error in the upgrade, so as of now we can keep this.